akka-cluster/src/main/scala/akka/cluster/CrossDcClusterHeartbeat.scala

```diff
@@ -219,9 +219,7 @@ private[cluster] final case class CrossDcHeartbeatingState(
    * Only the `nrOfMonitoredNodesPerDc`-oldest nodes in each DC fulfil this role.
    */
   def shouldActivelyMonitorNodes(selfDc: ClusterSettings.DataCenter, selfAddress: UniqueAddress): Boolean = {
-    /** Since we need ordering of oldests guaranteed, we must only look at Up (or Leaving, Exiting...) nodes */
-    def atLeastInUpState(m: Member): Boolean =
-      m.status != MemberStatus.WeaklyUp && m.status != MemberStatus.Joining
+    // Since we need ordering of oldests guaranteed, we must only look at Up (or Leaving, Exiting...) nodes
```

Comment:
yes, as far as I know (we skip joining and WeaklyUp because they don't have an upNumber yet)